remoteappmanager/db/orm.py

```diff
@@ -156,23 +167,60 @@ def transaction(session):
 def apps_for_user(session, user):
     """Returns a list of tuples, each containing an application and the
     associated policy that the specified orm user is allowed to run.
+    If the user is None, the default is to return an empty list.
 
     Parameters
     ----------
     session : Session
         The current session
     user : User
-        the orm User.
+        the orm User, or None.
 
     Returns
     -------
-    A list of tuples (orm.Application, orm.ApplicationPolicy)
+    A list of tuples (mapping_id, orm.Application, orm.ApplicationPolicy)
```

Comment:
The console script needs updating too, we could fix that in a separate issue
(skipped part of the traceback, as it is clear from the last line what we need:)
```
$ remoteappdb --db=~/remotemanagerapp.db user list
Traceback (most recent call last):
...
  File "/mnt/hgfs/VM_shared/SimPhoNy/simphony-remote/remoteappmanager/cli/remoteappdb/__main__.py", line 92, in <listcomp>
    for app, policy in orm.apps_for_user(session, user)]
ValueError: too many values to unpack (expected 2)
```